examples/fast_fourier_transformation/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022 Deepak Suresh
+
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if(NOT HPX_WITH_DISTRIBUTED_RUNTIME)
+  return()
+endif()
```

Comment:
The example uses local functionalities only (AFAICS), why do you require the distributed runtime to be available?

Reply:
Oh! I think it was by mistake I didn't comment out this one